renderer/index.css

```diff
@@ -113,6 +113,10 @@ table {
   opacity: 0.3;
 }
 
+.clickable {
+  cursor: pointer;
```

Comment:
Native apps don't use the hand pointer, only web pages do. This will make the app feel less native.